src/main/java/net/dv8tion/jda/api/entities/MessageEmbed.java

```diff
@@ -676,6 +678,32 @@ public String getUrl()
             return url;
         }
 
+        /**
+         * The url of the video, proxied by Discord
+         * <br>This url is used to access the video through Discord instead of directly to prevent ip scraping.
+         *
+         * @return Possibly-null String containing the proxied video url.
+         */
+        @Nullable
+        public String getProxyUrl()
+        {
+            return proxyUrl;
+        }
+
+        /**
+         * Returns an {@link AttachmentProxy} for this embed video.
+         *
+         * @return Possibly-null {@link AttachmentProxy} of this embed video
+         *
+         * @see    #getProxyUrl()
+         */
+        @Nullable
+        public AttachmentProxy getProxy()
+        {
+            final String proxyUrl = getProxyUrl();
+            return proxyUrl == null ? null : new AttachmentProxy(proxyUrl);
```

Comment:
I think this should be a `FileProxy`, videos proxied by Discord do not support resizing as far as I can tell.